src/vma/sock/sockinfo_udp.cpp

```diff
@@ -1371,7 +1371,7 @@ void sockinfo_udp::handle_recv_timestamping(struct cmsg_state *cm_state)
 	}
 
 	if (m_n_tsing_flags & SOF_TIMESTAMPING_RAW_HARDWARE) {
-		tsing.hwtimeraw = packet->path.rx.hw_timestamp;
+		packet->path.rx.p_ib_ctx_time_converter->convert_hw_time_to_system_time(packet->path.rx.hw_raw_timestamp, &tsing.hwtimeraw);
```

Comment:
@liranoz12 don't you want to keep the HW RAW TS value here?
and move this conversion to under the `if (SOF_TIMESTAMPING_SOFTWARE)`?